src/snowcli/cli/sql.py

```diff
@@ -62,5 +62,4 @@ def execute_sql(
         sql_text=sql,
         remove_comments=True,
     )
-    for result in results:
-        print_db_cursor(result)
+    print_output(OutputData(*results))
```

Comment:
Allow pass list of cursors

Reply:
I'm wondering about something more informative
```py
output = OutputData()
for cursors in cursors:
  output.add_cursor(cursor)
```

The result is probably the same but the code is more expressive. WDYT?

Reply:
nvm, I changed it to your idea